views/partials/inventory/product-order/search-grid.jade

```diff
@@ -11,8 +11,14 @@
             a(ui-sref="inventory.productOrder.detail({ productOrderId: productOrder.id })", ui-sref-opts="{ inherit: false }")
               time(title="{{ productOrder.requestedStartDate | amDateFormat: 'dddd, MMMM Do YYYY, h:mm a' }}") {{ productOrder.requestedStartDate | amCalendar }}
           .small
-            | by {{ productOrder.getCustomer().id }},
-            strong  {{ productOrder.state }}
+            dl.dl-horizontal
+              dt Customer
+              dd.text-dark-secondary {{ productOrder.getCustomer().id }}
+              dt Status
+              dd.text-dark-secondary
+                span {{ productOrder.state }}
+              dt Priority
+              dd.text-dark-secondary {{ productOrder.priority }}
```

Comment:
When this info is displayed it is too much indented to the right, it is almost in the middle of the panel header